<ui version="4.0" >
 <author>Palazhchenko Aleksey</author>
 <class>HistoryDlgBase</class>
 <widget class="QWidget" name="HistoryDlgBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>History</string>
  </property>
  <property name="locale" >
   <locale country="AnyCountry" language="C" />
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="0" column="0" >
    <widget class="QSplitter" name="splitter" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QLineEdit" name="collectionsSearchEdit" />
       </item>
       <item>
        <widget class="QTreeView" name="collectionsView" />
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2" >
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <property name="spacing" >
        <number>1</number>
       </property>
       <item>
        <widget class="QLineEdit" name="entriesSearchEdit" />
       </item>
       <item>
        <widget class="QTableView" name="entriesView" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX" >
   <number>10</number>
  </property>
  <property name="gridDeltaY" >
   <number>10</number>
  </property>
  <property name="gridSnapX" >
   <bool>true</bool>
  </property>
  <property name="gridSnapY" >
   <bool>true</bool>
  </property>
  <property name="gridVisible" >
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
